pallets/worker/src/tests.rs

```diff
@@ -680,3 +680,51 @@ fn should_check_for_generate_dataset() {
 
     assert_eq!(should_generate_dataset(block_number, stored_epoch), true);
 }
+
+#[test]
+fn add_block_2_rust_ethash() {
```

Comment:
@drewstone this test pass, and it using the `./data/2_rust_ethash.json` this the output json file from our `ethash` but I'm not sure if it should pass? maybe I'm testing it the wrong way.
can you test this out?

Reply:
Will check it out!